test/testFunction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require_once '../function.php';
+
+use PHPUnit\Framework\TestCase;
+
+class FunctionTest extends TestCase
+{
+    public function testGenerateRequestShouldReturnEverythingIfNoParameter()
```

Comment:
In PHPUnit we write

`testNameOfFunction_Scenario`

Reply:
ok fixed